multinode_installation/playbooks/install_ansible.yml

```diff
@@ -0,0 +1,13 @@
+--- # equivalent of 'install_ansible.sh'
+- hosts: all
+  sudo: true
+  tasks:
+    - name: add ansible ppa
+      apt_repository: repo='ppa:ansible/ansible'
```

Comment:
here is the relevant line of code: now we have ansible version 1.9 instead 1.5 on our vm